src/components/GenericComponent.tsx

```diff
@@ -0,0 +1,40 @@
+import { Button, Select, Text } from "@chakra-ui/react";
+import { SelectData } from "../utils/data";
+import { useGenericHook } from "../hooks/useGenericHook";
+
+//ten generic jest poprawny?
```

Comment:
ja nie widzę gdzie tego generyka T używasz.
GenericComponent i useGenericHook nic nie powiedzą następnemu programiście czym to jest

Reply:
nazwy:ok 

to jak zrobic z tego generica? jak to wzorcowo powinno wygladac?